app/src/main/res/layout/spell_detail_fragment.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
+
     <data>
```

Comment:
Remove this databinding stuff from all your layouts and set stuff in code instead

Especially remove this wherever its not used, like here